src/renderers/webgl/WebGLRenderLists.js

```diff
@@ -130,10 +130,10 @@ function WebGLRenderList() {
 
 	}
 
-	function sort() {
+	function sort( costumeOpaqueSort, costumeTransparentSort ) {
 
-		if ( opaque.length > 1 ) opaque.sort( painterSortStable );
-		if ( transparent.length > 1 ) transparent.sort( reversePainterSortStable );
+		if ( opaque.length > 1 ) opaque.sort( typeof costumeOpaqueSort === 'function' ? costumeOpaqueSort : painterSortStable );
```

Comment:
For performance reason, it is better to move
`typeof costumeOpaqueSort === 'function' ? ...`
into setOpaqueSort/setTransparentSort functions.

Reply:
Sure, good call!
I still have to check if `costumOpaqueSort` and `costumTransparentSort` are available for each sort iteration though.